packages/start-frontend/src/index.ts

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+// @ts-ignore
```

Comment:
Typescript is unable to find the prettier package, and is therefore throwing an error. I've disabled it for now, and I plan to fix it before merging to main.

Reply:
I thought that the package may be outdated, so I tried upgrading the prettier package (2.8.4 -> 3.2.5).

It resolved the the type error, but whenever I try to use `prettier.format`, it the following error occurs:

```
TypeError [ERR_INVALID_ARG_VALUE]: The argument 'filename' must be a file URL object, file URL string, or absolute path string. Received undefined
```

Still currently under investigation

Reply:
It was a build configuration issue. 😕
The code after building used `import.meta.url`, which is unavailable in CommonJS.
I added [shims: true](https://tsup.egoist.dev/#inject-cjs-and-esm-shims) in the tsup [config](https://github.com/monstar-lab-oss/ml-frontend/pull/1249/commits/1322a0d15ed0c69c2fffcadf82b9322d1534b5a3#diff-fcd0c36fd893decf59d5f94dda334f8e431f587a4d15f2898532955b63d4d40aR10).